### PR TITLE
Add emoji & sticker downloader

### PR DESCRIPTION
**Pull request by ToothyDev**

### CODE REVIEW

File: cogs/utility.py [hunk 1]
```diff
@@ -59,6 +61,21 @@ async def serverinfo(self, ctx):
...
         await ctx.respond(embed=embed)
 
+    @slash_command(brief="Get all server stickers & emojis!")
+    @discord.default_permissions(manage_guild=True)
+    @commands.cooldown(1, 600, commands.BucketType.user)
+    async def emoji_downloader(self, ctx):
+        await ctx.defer()
+        with zipfile.ZipFile('emoji_and_stickers.zip', 'w') as zipped_f:
+            for emoji in ctx.guild.emojis:
+                zipped_f.writestr(emoji.name + emoji.url[-4:], await emoji.read())
```
Comment (JoshuaSlui) [inline cogs/utility.py:73]: I'd like a delay between each emote download.

File: cogs/utility.py [hunk 2]
```diff
@@ -59,6 +61,21 @@ async def serverinfo(self, ctx):
...
+            for sticker in await ctx.guild.fetch_stickers():
+                async with aiohttp.ClientSession() as session:
+                    async with session.get(url=sticker.url) as response:
+                        zipped_f.writestr(sticker.name + ".png", await response.read())
```
Comment (JoshuaSlui) [inline cogs/utility.py:80]: You append a .png here, but what if the sticker is animated?

**Review by JoshuaSlui — CHANGES_REQUESTED** [review]

File: cogs/utility.py (hunk 2 above)
Comment (ToothyDev) [reply cogs/utility.py:80]: There is no specific .apng extension, APNG is basically built into PNG, just like .gif supports both animated and static images

File: cogs/utility.py (hunk 1 above)
Comment (ToothyDev) [reply cogs/utility.py:73]: Do you reckon half a second is enough? Else it might take a long time for the bot respond

Comment (JoshuaSlui) [reply cogs/utility.py:73]: One emote per second.

**Review by JoshuaSlui — APPROVED** [review]

**Pull request merged**